src/NuGet.Clients/NuGet.CommandLine/Commands/SignCommand.cs

```diff
@@ -190,4 +190,4 @@ private void ValidateCertificateInputs()
             }
```

Comment:
Undo changes to this file.

Reply:
Since `insert_final_newline = true` in NuGet.Client/.editorconfig. It adds a new line by default when I save it.

Reply:
Exclude this file from your PR.